src/components/multi-accordion-with-title/MultiAccordionWIthTitle.tsx

```diff
@@ -24,6 +25,8 @@ const MultiAccordionWithTitle: FC<MultiAccordionWithTitleProps> = ({
 }) => {
   const { t } = useTranslation()
 
+  const { isMobile } = useBreakpoints()

```

Comment:
if you use isMobile only for styles. remove it, and use brekpoints in styles, such as xs,sm etc

Reply:
oh, you are right, thank you)